lib/content-services/i18n/en.json

```diff
@@ -153,6 +153,16 @@
     },
     "ACTION": {
       "UNDO": "Undo"
+    },
+    "ERRORS": {
```

Comment:
@pionnegru can we please remove these strings from the ACA resources once it lands on ADF? So that we don't duplicate translators work